tsdb/db_test.go

```diff
@@ -2816,9 +2812,8 @@ func TestOpen_VariousBlockStates(t *testing.T) {
 
 	var loaded int
 	for _, l := range loadedBlocks {
-		if _, ok := expectedLoadedDirs[filepath.Join(tmpDir, l.meta.ULID.String())]; !ok {
-			t.Fatal("unexpected block", l.meta.ULID, "was loaded")
-		}
+		_, ok := expectedLoadedDirs[filepath.Join(tmpDir, l.meta.ULID.String())]
+		testutil.Assert(t, ok, "unexpected block", l.meta.ULID, "was loaded")
```

Comment:
Assert uses `t.Fatalf` inside it, so the message arguments should be similar to how you would pass to `fmt.Printf`. Can you make similar changes in other places where it is like this?
```suggestion
		testutil.Assert(t, ok, "unexpected block %s was loaded", l.meta.ULID.String())
```